keras/applications/imagenet_utils.py

```diff
@@ -58,6 +47,63 @@ def preprocess_input(x, data_format=None, mode='caffe'):
     return x
 
 
+def _preprocess_symbolic_input(x, data_format, mode):
+    global _IMAGENET_MEAN
+
+    if data_format is None:
+        data_format = K.image_data_format()
+    assert data_format in {'channels_last', 'channels_first'}
```

Comment:
Remove this assert (should be taken care of in the parent, and this is a private function)